completions/tmux

```diff
@@ -0,0 +1,134 @@
+# tmux(1) completion
+
+_tmux()
+{
+    local cur prev opts onePrev
+    _init_completion || return
+
+    COMPREPLY=()
+    cur="${COMP_WORDS[COMP_CWORD]}"
+    onePrev="${COMP_WORDS[COMP_CWORD-1]}"
+
+    if [ "$COMP_CWORD" -ge 2 ]; then
+        # Check if we want to list available sessions. We're going to assume
+        # this is defined as: [ls | list-session] -t .
+        local windowCommands currentSessions
+        windowCommands=("ls", "list-sessions")
+
+        prev="${COMP_WORDS[COMP_CWORD-2]}"
+        if [ "$prev" = "attach" ] || [ "$prev" = "attach-session" ] ; then
```

Comment:
Use words instead of COMP_WORDS and cword instead of COMP_CWORD. Also, rename this to something like preprev instead of overloading the usual definition of prev